_inc/client/plans/plan-grid.jsx

```diff
@@ -41,6 +41,11 @@ class PlanGrid extends React.Component {
 
 		return (
 			<div className="plan-features">
+				<div className="plans-mobile-notice dops-card">
+					<h2>Your plan</h2>
+					<p>You're currently on the Jetpack [plan name].</p>
```

Comment:
This line actually needs the plan name now, and the entire block needs translated strings.